### PR TITLE
add kafkaTemplate to Lep context

### PR DESCRIPTION
**Pull request by ArturRoze**

### CODE REVIEW

File: src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java [hunk 1]
```diff
@@ -79,6 +81,7 @@ protected void bindExecutionContext(final ScopedContext executionContext) {
 
         Map<String, Object> templates = new HashMap<>();
         templates.put(BINDING_SUB_KEY_TEMPLATE_REST, restTemplate);
+        templates.put(BINDING_SUB_KEY_TEMPLATE_KAFKA, kafkaTemplate);
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java:84]: pls add field to LepContext class

File: src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java [hunk 2]
```diff
@@ -52,6 +53,7 @@ public class XmUaaLepProcessingApplicationListener extends SpringLepProcessingAp
     private final TenantPropertiesService tenantPropertiesService;
     private final LepTokenGranter lepTokenGranter;
     private final ClientDetailsService clientDetailsService;
+    private final KafkaTemplate kafkaTemplate;
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java:56]: better to use KafkaTemplateService

File: src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java (hunk 1 above)
Comment (ArturRoze) [reply src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java]: Done

File: src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java (hunk 2 above)
Comment (ArturRoze) [reply src/main/java/com/icthh/xm/uaa/lep/XmUaaLepProcessingApplicationListener.java]: Done

**Review by sergeysenja1992 — APPROVED** [review]

**Review by major13ua — APPROVED** [review]

**Review by pkev777 — APPROVED** [review]

**Pull request merged**